apps-rendering/src/components/liveBlocks.tsx

```diff
@@ -44,18 +45,18 @@ const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format }) => {
 						<footer
 							css={css`
 								display: flex;
-								justify-content: space-between;
+								justify-content: end;
 							`}
 						>
 							{block.lastModified.kind === OptionKind.Some &&
 								block.firstPublished.kind === OptionKind.Some &&
 								block.lastModified.value >
 									block.firstPublished.value && (
 									<LastUpdated
-										lastUpdated={Number(
+										lastUpdated={block.lastModified.value}
+										lastUpdatedDisplay={formatUTCTimeDateTz(
```

Comment:
It was agreed with Jamie B and David F that for apps-rendering we'd render UTC on the server